### PR TITLE
Unified About: App Logos

### LINKED ISSUE #17405: [iOS] About Screen v1: Add logo cell
Add cell showing logos for all Automattic apps. Ideally dynamic based on device motion.

Screenshot 2021-11-02 at 11 08 38

### PR DESCRIPTION
**Pull request by frosty**
Fixes #17405. This PR adds a cell containing logos of the various Automattic apps. This is rendered with SpriteKit and the apps randomly fall into place using physics bodies. It should also handle dark mode correctly. Originally the logos responded to the motion of the user's device but I can't currently find a way to make this work nicely when the app can rotate to any orientation, so I've taken the code out for now.

| Light Mode | Dark Mode |
|---|---|
| Simulator Screen Shot - iPhone 12 Pro - 2021-11-04 at 16 10 59 | Simulator Screen Shot - iPhone 12 Pro - 2021-11-04 at 16 12 49 |

On wider devices like Max phones or iPad, I've restricted the maximum width of the SpriteKit area, otherwise things were looking very sparse with a few balls spread out in a line over a wide distance:

| Before | After |
|---|---|
| image (4) | Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2021-11-04 at 13 29 24 |

**To test**

* Build and run
* Navigate to about, check the logos look good. Try tapping on some too, they bounce a little.
* Test on a range of device sizes, rotate the screen, and check light and dark mode.

## Regression Notes

1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Me/App Settings/About/UnifiedAboutViewController.swift [hunk 1]
```diff
@@ -28,6 +28,33 @@ struct AboutItem {
         }
     }
 
+    var cellHeight: CGFloat {
+        switch cellStyle {
+        case .appLogos:
+            return 150.0
```
Comment (diegoreymendez) [inline WordPress/Classes/ViewRelated/Me/App Settings/About/UnifiedAboutViewController.swift]: Consider moving this value into a constant.

File: WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift [hunk 2]
```diff
@@ -253,6 +253,10 @@ class MeViewController: UITableViewController {
 
     private func pushAbout() -> ImmuTableAction {
         return { [unowned self] _ in
+            defer {
+                self.tableView.deselectSelectedRowWithAnimation(true)
+            }
```
Comment (diegoreymendez) [inline WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift]: Why not execute this directly at the bottom of this code block to avoiding the extra `defer`  here?

**Review by diegoreymendez — APPROVED** [review]
Just added two comments for your consideration.  The PR works as described and the code looks good.

File: WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift (hunk 2 above)
Comment (frosty) [reply WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift]: I was just replicating the method below, but I think I prefer it in the completion block for the presentation. 👍

**Pull request merged**